powersimdata/network/usa_tamu/constants/zones.py

```diff
@@ -30,7 +30,7 @@
 mappings = {"loadzone", "state", "state_abbr", "interconnect"}
 
 # Define combinations of interconnects
-interconnect_combinations = {"USA", "Texas_Western"}
+interconnect_combinations = {"USA": 3, "Texas_Western": 2}
```

Comment:
Would it be more useful to have a list of interconnects as values, i.e..:
```
interconnect_combinations = {"USA": ["Eastern", "Texas", "Western"], "Texas_Western": ["Texas", "Western"]}
```

Reply:
I think that's probably cleaner to someone looking at just this file, yeah.

Reply:
Done.